src/main/java/slimeknights/mantle/network/packet/OpenNamedBookPacket.java

```diff
@@ -35,8 +35,8 @@ public void handleThreadsafe(NetworkEvent.Context context) {
     }
   }
 
-  static class ClientOnly {
-    static void errorStatus(ResourceLocation book) {
+  public static class ClientOnly {
+    public static void errorStatus(ResourceLocation book) {
```

Comment:
I usually prefer to keep these clients only methods specific to the packet to prevent any risk with class loading. Probably best to just move the logic to the book command class, though I suppose for safety you'd have to keep client only around here to call the book command helper.